src/shared/SelectInput/SelectInput.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const SelectInput = ({ label, options, value, onChange, ...props }) => {
+  return (
+    <div className="form-group">
+      {label && <label>{label}</label>}
+      <select className="form-control" value={value} onChange={onChange} {...props}>
+        {options.map((option) => (
+          <option key={option.value} value={option.value}>
+            {option.label}
+            {option.additionalInfo && (
+              <span> - {option.additionalInfo}</span>
```

Comment:
My browser gives a warning about using `span` inside `option`: `Warning: validateDOMNesting(...): <span> cannot appear as a child of <option>.`